framework/src/meshgenerators/FileMeshGenerator.C

```diff
@@ -87,6 +95,9 @@ FileMeshGenerator::generate()
     {
       if (mesh->processor_id() == 0)
       {
+        if (getParam<bool>("discontinuous_spline_extraction"))
+          exreader->set_discontinuous_bex(true);
```

Comment:
what do we get if the user does not set this parameter properly on a mesh that has these features?

Reply:
It depends.  ;-)

If we have an input mesh for which the assembly elements form a conforming mesh, and all our IGA constraints are implicit in the IGA input mesh, then we get the same FE space, but with fewer constraint equations from the IGA level, because the C0-continuity level of that constraint is maintained by elements simply sharing nodes rather than by redundant equations for each "copy" of a common node.  Global solves on that FE space give the same results to within FP error; any local operations projected back to the spline nodes only give the same results to within FE approximation error (the pseudoinverse involved in that projection minimizes an algebraic norm rather than a nice basis-independent Sobolev norm).

If we have an input mesh for which the assembly elements get IGA constraints from an explicit "extraction operator" (not in this PR, but coming in the next), then when we attempt to load the extraction operator you get a nice message about the numbering not matching.

If we have an input mesh for which the assembly elements form a conforming mesh, and all our IGA constraints are implicit in the IGA input mesh ... I don't actually recall what happens.  I had to put this down for too long to work on other things, and now I can't find a mesh fitting that case.  IIRC you get a somewhat confusing error message at the libMesh level.

I've asked Coreform for a new mesh like that, and if they don't have anything handy I'll try cooking one up myself to see if the failure is user-friendly enough or fix it if not.  In the "not" case, any changes will have to be at the libMesh level, not here.